config/locales/en.yml

```diff
@@ -857,6 +857,8 @@ en:
     introduction:
       unspecified: "If you have a question about the Manage training for early career teachers service, you can contact the DfE using this form."
       change-participant-lead-provider: "Complete this form to request a change to %{participant_name}’s registered training provider."
+      change-cohort-lead-provider: "Complete this form to request a change to %{cohort_year} cohort’s registered training provider."
+      change-cohort-delivery-partner: "Complete this form to request a change to %{cohort_year} cohort’s delivery partner."
```

Comment:
Needs a content review from @Lloyd-K - not sure we should use "cohort"?

Reply:
Can we use this format to give schools more context? 

Complete this form to tell us about a change of provider for ECTs and mentors who started their training in the 2023 to 2024 academic year.

Reply:
I've swapped this over.